support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -21,6 +26,21 @@ export const productCatalogDescription: Record<string, ProductDescription> = {
 				tooltip: `Guardian Weekly is a beautifully concise magazine featuring a handpicked selection of in-depth articles, global news, long reads, opinion and more. Delivered to you every week, wherever you are in the world.`,
 			},
 		],
+		ratePlans: {
+			MonthlyWithGuardianWeekly: {
+				billingPeriod: 'Month',
```

Comment:
In the long term, I'd like to try and get this into [the API](https://product-catalog.guardianapis.com/product-catalog.json).

It needs a bit of thinking though as the `billingPeriod` is actually part of the `charge` which we don't really use in the UI.

I think we might be able to determine this value as in practise all `charges` for our `ratePlans` have the same `billingPeriod` so we could do some sort of validation on that, and then hoist the property up to the ratePlan if it works.

Reply:
[Here's a discussion](https://github.com/guardian/support-frontend/discussions/5919)